litellm/utils.py

```diff
@@ -2464,15 +2464,7 @@ def function_setup(
                 f"[Non-Blocking] {traceback.format_exc()}; args - {args}; kwargs - {kwargs}"
             )
             raise e
-
```

Comment:
I was trying to reverse the changes made [here](https://github.com/BerriAI/litellm/commit/9b7383ac67269841c9e4c27e7144eb9a56c79029), since this did not fix the issue. The changes in `utils.py` can be safely ignored without any problems since the code for `acompletion` function was updated in `main.py`.

Reply:
Can you undo the changes you made in utils.py @JohnnyRacer